src/test/java/teammates/test/cases/browsertests/FeedbackMcqQuestionUiTest.java

```diff
@@ -291,4 +294,294 @@ public void testDeleteQuestionAction() {
+        assertFalse(browser.driver.findElement(By.id("mcqOtherOptionFlag-" + NEW_QUESTION_INDEX)).isSelected());
+
+        // Assign Other option and test that other weight cell is displayed.
+        feedbackEditPage.clickAddMcqOtherOptionCheckboxForNewQuestion();
+        assertTrue(feedbackEditPage.getMcqOtherWeightBox(NEW_QUESTION_INDEX).isDisplayed());
+
+        // Uncheck checkboxes for consistency among other tests,
+        // otherwise these settings will persist after cancelling the question form
+        // Uncheck the 'Choices are weighted' checkbox.
+        feedbackEditPage.clickMcqAssignWeightCheckboxForNewQuestion();
+        feedbackEditPage.clickAddMcqOtherOptionCheckboxForNewQuestion();
+
+        // Cancel question
+        feedbackEditPage.clickDiscardChangesLinkForNewQuestion();
+        feedbackEditPage.waitForConfirmationModalAndClickOk();
+    }
+
+    @Test
+    public void testMcqWeightsFeature_shouldHaveCorrectDefaultValue() throws Exception {
+
+        ______TS("MCQ: Weight cells are hidden by default");
+        feedbackEditPage.clickAddQuestionButton();
+        feedbackEditPage.selectNewQuestionTypeAndWaitForNewQuestionPanelReady("MCQ");
+        // Check if the 'Choices are weighted' checkbox unchecked by default.
+        assertFalse(feedbackEditPage.isMcqAssignWeightCheckboxChecked(NEW_QUESTION_INDEX));
+        // Check if the MCQ weights column hidden by default
+        assertFalse(feedbackEditPage.getMcqWeightsColumn(NEW_QUESTION_INDEX).isDisplayed());
+        assertFalse(feedbackEditPage.getMcqOtherWeightBox(NEW_QUESTION_INDEX).isDisplayed());
+
+        ______TS("MCQ: Check Mcq weight default value");
+        feedbackEditPage.fillQuestionTextBox("MCQ weight feature", NEW_QUESTION_INDEX);
+        feedbackEditPage.fillQuestionDescription("More details", NEW_QUESTION_INDEX);
+        feedbackEditPage.clickMcqAssignWeightCheckboxForNewQuestion();
+        feedbackEditPage.clickAddMcqOtherOptionCheckboxForNewQuestion();
+
+        // Fill MCQ choices and check corresponding weight values
+        feedbackEditPage.fillMcqOptionForNewQuestion(0, "Choice 1");
+        feedbackEditPage.verifyFieldValue("mcqWeight-0--1", "0");
+        feedbackEditPage.fillMcqOptionForNewQuestion(1, "Choice 2");
+        feedbackEditPage.verifyFieldValue("mcqWeight-1--1", "0");
+
+        // Verify MCQ other weight default value
+        feedbackEditPage.verifyFieldValue("mcqOtherWeight--1", "0");
+        feedbackEditPage.clickAddQuestionButton();
+        feedbackEditPage.waitForTextsForAllStatusMessagesToUserEquals(Const.StatusMessages.FEEDBACK_QUESTION_ADDED);
+        assertNotNull(BackDoor.getFeedbackQuestion(courseId, feedbackSessionName, 1));
+
+        // verify html page
+        feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackMcqQuestionWeightAddSuccess.html");
+
+        // Delete the question
+        feedbackEditPage.clickDeleteQuestionLink(1);
```

Comment:
As all the weight tests are on separate `@Test`, It is hard to predict which tests run after which tests, so if I leave the question without deleting it, I will have to change the question numbering (for IDs or for asserts whatever) on every test case in the whole file and It may bring a lot of inconsistencies due to the unpredictableness of the execution order of tests. SO, I just delete each question after a test completes to not hamper the whole test suite, and so that we can easily add more tests later without thinking about the inter-dependencies of all the previous tests.